src/wdio/services/TerraService.js

```diff
@@ -37,21 +46,41 @@ function matchReference() {
 * @param sizes - [String] of viewport sizes.
 * @return [Object] of viewport sizes.
 */
-const viewports = (...sizes) => {
-  const widths = {
-    tiny: { width: 470, height: 768 },
-    small: { width: 622, height: 768 },
-    medium: { width: 838, height: 768 },
-    large: { width: 1000, height: 768 },
-    huge: { width: 1300, height: 768 },
-    enormous: { width: 1500, height: 768 },
-  };
+const getViewports = (...sizes) => {
+  let viewportSizes = Object.keys(VIEWPORTS);
+  if (sizes.length) {
+    viewportSizes = sizes;
+  }
+  return viewportSizes.map(size => VIEWPORTS[size]);
+};
 
-  if (sizes.length === 0) {
-    return global.viewport('tiny', 'small', 'medium', 'large', 'huge');
+const resize = (...args) => {
+  const test = args.slice(-1)[0];
+  const viewportSizes = args.slice(0, -1);
+
+  if (typeof test !== 'function') {
+    throw new Error('The last argument must be a function');
   }
 
-  return sizes.map(size => widths[size]);
+  viewportSizes.forEach((name) => {
+    global.browser.setViewportSize(VIEWPORTS[name]);
+    test(name, VIEWPORTS[name]);
+  });
+};
+
+const should = {
+  beAccessible(options) {
+    global.it('is accessible', () => {
+      global.expect(global.browser.axe(options)).to.be.accessible();
+    });
+  },
+
+  beComparable({ name = 'default', css = '[data-reactroot]', viewports = {} }) {
```

Comment:
A few things:
- IMO `matchScreenshot` or `matchSnapshot` better describes this helper. `beComparable` could be interpreted differently and I feel like we want an explicit name for a test helper
- How would you feel about breaking name and css out of the param obj? name is specific to our helper and then css and options would be more consistent with visual regression's checkElement func allowing more flexibility to pass in more than viewports
- I feel like the term `element` or `selector` might be clearer than `css`

Reply:
> How would you feel about breaking name and css out of the param obj? name is specific to our helper and then css and options would be more consistent with visual regression's checkElement func allowing more flexibility to pass in more than viewports

Yeah I had that at first but the problem then becomes if you want to change the viewports but not change the name or default element....  you are now forced to reset them. Thoughts?

Reply:
hmm what about something like:
```
beComparable(name = 'default', { css = '[data-reactroot]', options  = { viewports = {} } })
```
or
```
beComparable(name = 'default', { css = '[data-reactroot]',  viewports = {}, options  = {})
// then merge options & viewports
```
side note: Should we default to `viewports = VIEWPORTS`?

Reply:
```
beComparable(name = 'default', { css = '[data-reactroot]',  viewports = {}, options  = {})
beComparable(name = 'default')
beComparable({ css = '[data-reactroot]',  viewports = {}, options  = {})
```

I wonder how hacky it is to support all 3? Rules would be as follows:

1. 1st param can be object or String. If string then its name, else its options
2. 2nd param, if present is always assumed to be options.

Thoughts?